app/views/admin/categories/index.html.erb

```diff
@@ -8,15 +8,17 @@
         <tr>
             <th>Title</th>
             <th>Description</th>
-            <th>Action</th>
+            <th colspan="3">Action</th>
         </tr>
     </thead>
     <tbody>
         <% @categories.each do |category| %>
             <tr>
-                <td><%= category.title %></td>
+                <td><%= link_to category.title, admin_category_url(category) %></td>
```

Comment:
@CharlotteDS Use `path` helpers in view pages
example: admin_category_url -> admin_category_path